.github/scripts/bump-nuget.py

```
@@ -0,0 +1,133 @@
+
+    if skipped_third_party:
+        print()
+        print(f"Skipped {len(skipped_third_party)} third-party package(s):")
+        print("\n".join(skipped_third_party[:5]))  # Show first 5
+        if len(skipped_third_party) > 5:
+            print(f"  ... and {len(skipped_third_party) - 5} more")
+
+    with open("Directory.Packages.props", "w") as f:
+        f.write(new_content)
+
+    return 0 if changes else 0  # Return 0 even if no changes (not an error)
```

Ruff (0.15.1)

[warning] Useless `if`-`else` condition

(RUF034)